dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:8-alpine AS build
+
+# Create the app directory
+WORKDIR /app
+
+# Copy package json
+COPY package*.json ./
+
+# Install dependencies
+RUN yarn install
+
+COPY . .
+
+FROM build as publish
+RUN yarn build
+
+FROM node:8-alpine AS runtime
+WORKDIR /app
+
+COPY --from=publish /app/package*.json ./
+COPY --from=publish /app/yarn.lock .
```

Comment:
Remove this line. You don't use yarn and the there is no yarn.lock file!

Reply:
fixed